keps/sig-cloud-provider/20191004-out-of-tree-credential-providers.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Out-of-Tree Credential Providers
+authors:
+  - "@mcrute"
+  - "@nckturner"
+owning-sig: sig-cloud-provider
+participating-sigs:
+  - sig-node
+  - sig-auth
+reviewers:
+  - "@andrewsykim"
+  - "@cheftako"
+  - "@tallclair"
+  - "@mikedanese"
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2019-10-04
+last-updated: 2019-12-10
+status: provisional
```

Comment:
I think there's enough details to make this `implementable`

Reply:
Also it needs to be implementable if we are going to start making code changes for it in 1.18